client/src/models/trees.js

```diff
@@ -9,118 +8,147 @@ const trees = {
+        .then(res => {
+          this.getTree(res.data);
+        })
+        .catch(err =>
+          console.error(`ERROR: FAILED TO GET SELECTED TREE ${err}`)
+        );
     },
     async getLocationName(payload, rootState) {
-      if ((rootState.trees.byId[payload.id] &&
-        rootState.trees.byId[payload.id].location &&
-        rootState.trees.byId[payload.id].location.lat !== payload.lat &&
-        rootState.trees.byId[payload.id].location.lon !== payload.lon) ||
-        (
-          !rootState.trees.byId[payload.id] || !rootState.trees.byId[payload.id].location
-        )) {
-        const query = `https://nominatim.openstreetmap.org/reverse?format=jsonv2&lat=${payload.latitude}&lon=${payload.longitude}`
-        Axios.get(query)
-          .then((response) => {
-            this.receiveLocation(response.data, payload)
-          })
+      if (
+        (rootState.trees.byId[payload.id] &&
+          rootState.trees.byId[payload.id].location &&
+          rootState.trees.byId[payload.id].location.lat !== payload.lat &&
+          rootState.trees.byId[payload.id].location.lon !== payload.lon) ||
+        (!rootState.trees.byId[payload.id] ||
+          !rootState.trees.byId[payload.id].location)
+      ) {
+        const query = `https://nominatim.openstreetmap.org/reverse?format=jsonv2&lat=${
+          payload.latitude
+        }&lon=${payload.longitude}`;
+        Axios.get(query).then(response => {
+          this.receiveLocation(response.data, payload);
+        });
       } else {
-        this.receiveLocation(null, { id: payload.id, address: 'cached' })
+        this.receiveLocation(null, { id: payload.id, address: 'cached' });
       }
     },
     async markInactiveTree(id) {
       const query = `${API_ROOT}/trees/${id}/`;
-      const data = { "active": false };
-      Axios.patch(query, data)
-        .then((response) => {
-          this.receiveStatus(response.status);
-        })
-    },
-    async showTree(id) {
-
+      const data = { active: false };
+      Axios.patch(query, data).then(response => {
+        this.receiveStatus(response.status);
+      });
     },
+    async showTree(id) {},
     async sortTrees(payload, rootState) {
       const { page, rowsPerPage, order } = rootState.trees;
-      const newOrder = (order === 'asc') ? 'desc' : 'asc';
-      const query = `${API_ROOT}/trees?filter[order]=${payload.orderBy} ${newOrder}&filter[limit]=${rowsPerPage}&filter[skip]=${page * rowsPerPage}&filter[fields][lat]=true&filter[fields][lon]=true&filter[fields][id]=true&filter[fields][timeCreated]=true&filter[fields][timeUpdated]=true`;
-      Axios.get(query)
-        .then((response) => {
-          this.getTrees(response.data, { page: page, rowsPerPage: rowsPerPage, orderBy: payload.orderBy, order: newOrder });
+      const newOrder = order === 'asc' ? 'desc' : 'asc';
+      const query = `${API_ROOT}/trees?filter[order]=${
+        payload.orderBy
```

Comment:
Formatting is kinda strange... I think you should keep the interpolated values on the same line, and keep the width about the same, but this is more of a nit